docs/external-cluster.md

```diff
@@ -0,0 +1,82 @@
+# Use External Virtink Infrastructure Cluster
+
+External Virtink infrastructure cluster indicates that the Cluster API management and Virtink components are deployed in different Kubernetes clusters, which is the recommended mode. Before referring [quick started guide](https://cluster-api.sigs.k8s.io/user/quick-start.html) for launching a cluster on Virtink, the following requirements should be met.
+
+## Access Virtink Cluster By Kubeconfig
```

Comment:
You don't need to cap words like "by" or "in". You can refer to https://capitalizemytitle.com/style/APA/ for how to cap titles